README.md

```diff
@@ -31,6 +31,7 @@ Currently Minigalaxy can be displayed in the following languages:
 - English
 - Dutch
 - Taiwanese Mandarin
+- Norwegian Bokmål
```

Comment:
Could this be left out for now? I'd like the README to reflect the currently released version. Once the next release goes live, I'll add it again.

Reply:
Done.